crates/motsu/src/context.rs

```diff
@@ -538,6 +538,17 @@ impl VMContext {
     fn router(self, address: Address) -> VMRouterContext {
         VMRouterContext::new(self.thread_id, address)
     }
+
+    /// Get the current chain ID.
+    pub fn chain_id(self) -> U256 {
```

Comment:
```suggestion
    pub(crate) fn chain_id(self) -> U256 {
```
should only be available within the crate